Dev container: Fresh dev container fails to install dependencies

### Describe the bug

Dev container fails to install dependencies. 

### Expected behavior

Fresh dev container to boot successfully.

### Steps to reproduce the issue

Load the dev container following the [README instructions](https://github.com/eclipsesource/jsonforms#vs-code-dev-container)

### Screenshots

_No response_

### In which browser are you experiencing the issue?

-

### Which Version of JSON Forms are you using?

master branch

### Framework

_No response_

### RendererSet

_No response_

### Additional context

- Could be related to https://github.com/eclipsesource/jsonforms/issues/2049#issue-1446981982


<details><summary>Log</summary>
<p>


```text
Running the postCreateCommand from devcontainer.json...

[12366 ms] Start: Run in container: /bin/sh -c npm ci && npm run init && npm run build
npm WARN prepare removing existing node_modules/ before installation

> fsevents@1.2.13 install /workspaces/jsonforms/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js


Skipping 'fsevents' build as platform linux is not supported

> core-js@3.26.1 postinstall /workspaces/jsonforms/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js:
> https://opencollective.com/core-js 
> https://patreon.com/zloirock 
> bitcoin: bc1qlea7544qtsmj2rayg0lthvza9fau63ux0fstcz 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> core-js@2.6.12 postinstall /workspaces/jsonforms/node_modules/empower-core/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> core-js@2.6.12 postinstall /workspaces/jsonforms/node_modules/call-matcher/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@2.6.12 postinstall /workspaces/jsonforms/node_modules/espurify/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@2.6.12 postinstall /workspaces/jsonforms/node_modules/babel-plugin-espower/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

added 1582 packages in 68.045s

> jsonforms-monorepo@ init /workspaces/jsonforms
> lerna bootstrap --hoist

lerna notice cli v3.22.1
lerna info Bootstrapping 16 packages
lerna info Installing external dependencies
lerna info hoist Installing hoisted dependencies into root
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna ERR! Error: Command failed: npm install
lerna ERR! 
lerna ERR!     at makeError (/workspaces/jsonforms/node_modules/@lerna/child-process/node_modules/execa/index.js:174:9)
lerna ERR!     at /workspaces/jsonforms/node_modules/@lerna/child-process/node_modules/execa/index.js:278:16
lerna ERR!     at runMicrotasks (<anonymous>)
lerna ERR! lerna Command failed: npm install
lerna ERR! lerna 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! jsonforms-monorepo@ init: `lerna bootstrap --hoist`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the jsonforms-monorepo@ init script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2023-06-13T15_31_06_075Z-debug.log
[272836 ms] postCreateCommand failed with exit code 1. Skipping any further user-provided commands.
Done. Press any key to close the terminal.
```

</p>
</details>

--- a/.devcontainer/Dockerfile
+++ b/.devcontainer/Dockerfile
@@ -2,11 +2,13 @@
 
 # [Choice] Node.js version: 16
 ARG VARIANT="16"
-FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${VARIANT}
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:1-${VARIANT}
 
-# [Optional] Uncomment this section to install additional OS packages.
-# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-#     && apt-get -y install --no-install-recommends <your-package-list-here>
+# Install chromium and point the CHROME_BIN environment variable to it
+# This allows running angular-material's tests in the dev container
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+  && apt-get -y install --no-install-recommends chromium
+ENV CHROME_BIN=/usr/bin/chromium
 
 # [Optional] Uncomment if you want to install an additional version of node using nvm
 # ARG EXTRA_NODE_VERSION=10

--- a/README.md
+++ b/README.md
@@ -43,6 +43,9 @@ As an alternative to the first time setup, you can use the provided [VS Code dev
 - Wait until the container is built and loaded
 - First time setup and an initial build of all packages has been executed in the container
 
+**Note:** If you have installed dependencies before opening the remote container, its initialization might fail.
+In this case, you can try to clean the repository with `git clean -dfx`. Beware that this removes all untracked files!
+
 ### Build & Testing
 
 - Build (all packages): `pnpm run build`

